ckanext/ontario_theme/fanstatic/common.css

```diff
@@ -29,4 +29,110 @@
 .toolbar .breadcrumb .active a,
 .toolbar .breadcrumb a.active {
   font-weight: normal;
+}
+/* ========================================================================
+   Page Alerts -Design System
+   ======================================================================== */
+.ontario-alert{
+    border-width:0 0 0 0.9375rem;
+    border-style:solid;
+    padding:1.5rem;
+    margin:2rem 0 2.5rem 0;
+}
+.ontario-alert__header{
+    display:flex
+}
+.ontario-alert__header-icon{
+    display:inline-block;
+    vertical-align:top;
+    margin-right:.95rem;
+}
+.ontario-alert__header-icon svg{
+    width:36px;
+    height:36px
+}
+@media screen and (max-width: 40em){
+    .ontario-alert__header-icon svg{
+        width:32px;
+        height:32px
+    }
+}
+.ontario-alert__header-title{
+    margin-bottom:.75rem;
+    padding-top:2px
+}
+.ontario-alert__body{
+    margin-left:3.25rem;
+    max-width:48rem;
+}
+@media screen and (max-width: 40em){
+    .ontario-alert__body{
+        margin-left:0
+    }
+}
+.ontario-alert__body p{
+    margin-top:0;
+    font-size: 16px;
+}
+.ontario-alert__body p:last-of-type{
+    margin-bottom:.5rem
+}
+.ontario-alert--informational{
+    border-color:#1080a6;
+    background-color:#e2f0f4
+}
+.ontario-alert--informational svg{
```

Comment:
Why include an `svg` class?

Reply:
The icons we should be using are `svg`s , I added a comment on line 14 of `ontario_theme_contact_us.html`, but I can add a comment above line 10 and in the css

Reply:
But they are not svg. There is no svg in the code so I recommend removing this class until we need it.